fearless/Common/Storage/SubstrateDataModel.xcdatamodeld/SubstrateDataModel.xcdatamodel/contents

```diff
@@ -19,8 +19,13 @@
     <entity name="CDRuntimeMetadataItem" representedClassName="CDRuntimeMetadataItem" syncable="YES" codeGenerationType="class">
         <attribute name="identifier" optional="YES" attributeType="String"/>
         <attribute name="metadata" optional="YES" attributeType="Binary"/>
+        <attribute name="txVersion" optional="YES" attributeType="Integer 32" defaultValueString="0" usesScalarValueType="YES"/>
         <attribute name="version" optional="YES" attributeType="Integer 32" defaultValueString="0" usesScalarValueType="YES"/>
     </entity>
+    <entity name="CDSingleValue" representedClassName="CDSingleValue" syncable="YES" codeGenerationType="class">
+        <attribute name="identifier" attributeType="String"/>
+        <attribute name="payload" attributeType="Binary"/>
+    </entity>
```

Comment:
How is it different from CDChainStorageItem?

Reply:
This is for cache. Now it is used to cache price but can be used to cache any single value stuff. CDChainStorage is used to cache blockchain stuff and looks like it will evolve. I see that we need to move network type to separate field

Reply:
Thanks for the explanation